src/msw/webview_ie.cpp

```diff
@@ -857,6 +857,67 @@ wxString wxWebViewIE::GetPageText() const
     }
 }
 
+wxWebViewIEEmulationLevel wxWebViewIE::GetEmulationLevel()
+{
+    wxRegKey key(wxRegKey::HKCU, _T("SOFTWARE\\Microsoft\\Internet Explorer\\Main\\FeatureControl\\FEATURE_BROWSER_EMULATION"));
+    if ( key.Exists() )
+    {
+        long val = 0;
+
+        if ( !key.QueryValue(wxGetFullModuleName().AfterLast('\\'), &val) )
+        {
+            wxLogWarning(_("Can not get level value of registry key \
+                SOFTWARE\\Microsoft\\Internet Explorer\\Main\\FeatureControl\\FEATURE_BROWSER_EMULATION"));
+            return wxWEBVIEWIE_EMULATION_LEVEL_EMPTY;
```

Comment:
In this case can we not just return `wxWEBVIEWIE_EMULATION_LEVEL_7_DEFAULT` as that is the value when unset?